etiqet-core/src/main/java/com/neueda/etiqet/fixture/EtiqetHandlers.java

```diff
@@ -623,6 +627,32 @@ public void createMessageForClient(String msgType, String clientName, String mes
         messageMap.put(messageName, message);
     }
 
+    public void createMessageNoAlias(String msgType, String clientName, String params) throws EtiqetException {
```

Comment:
Please try to avoid code repetition. Use aux method to group common behaviour improving maintainability